ui/src/app/job-list/job-list.component.ts

```diff
@@ -166,7 +169,9 @@ export class JobListComponent implements OnInit {
   }
 
   private onClientPaginate(e: PageEvent) {
-    this.settingsService.setPageSize(e.pageSize, this.projectId);
+    if (e.pageSize != this.pageSize) {
+      this.settingsService.setPageSize(e.pageSize, this.projectId);
```

Comment:
This should also update `this.pageSize`, else you won't save if the user switches back to 50

Reply:
- User changes page size to 100
-> this saves
- User changes page size back to 50
-> this doesn't save (`this.pageSize` is still 50)

Reply:
ok, all set